app/controllers/locales_controller.rb

```diff
@@ -1,26 +1,29 @@
 class LocalesController < ApplicationController
-  before_action :check_permission, only: [:new, :create, :update, :edit]
-
-  def check_permission
-    logged_in_as_admin? || permit?("translation_admin") || access_denied
-  end
 
   def index
+    authorize Locale
+
     @locales = Locale.default_order
   end
 
   def new
+    authorize Locale
```

Comment:
For new, edit, update, and so on, we generally prefer to call `authorize` on the instance variable, not the class.

Reply:
done